packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -14,12 +14,12 @@
   "pci_database_add_step5_title": "Configurez vos options",
   "pci_database_add_name": "Nommez votre service de base de données",
   "pci_database_add_network_mode_label": "Type de réseau",
-  "pci_database_add_network_mode_public": "Public",
-  "pci_database_add_network_mode_private": "Privé",
+  "pci_database_add_network_mode_public": "Réseau public (Internet) ",
+  "pci_database_add_network_mode_private": "Réseau privé (vRack)",
   "pci_database_add_network_mode_public_incompatible": "L'offre sélectionnée n'est pas compatible avec les réseaux publics.",
   "pci_database_add_network_mode_private_incompatible": "L'offre sélectionnée n'est pas compatible avec les réseaux privés.",
   "pci_database_add_privateNetwork_label": "Réseau privé attaché",
-  "pci_database_add_privateNetwork_description": "Par défaut, les bases de données possèdent un réseau public. Vous pouvez également ajouter un réseau privé, mais vous perdrez la configuration que vous venez d’établir.",
+  "pci_database_add_privateNetwork_description": "Seuls les services (serveurs dédiés, Vms, clusters K8s, …) connectés a ce {{htmlBoldTag}}réseau privé (vRack){{htmlBoldTagEnd}} pourront communiquer avec votre base de données. ",
```

Comment:
Good catch ! 
It used to be plain html `<b>réseau privé (vRack)</b>`, but I had this comment to avoid html tags: https://github.com/ovh/manager/pull/7460#discussion_r927484917. I probably forgot to include the html in the commit :confused: 
However, the solution in the comment does not seem to work, the closing tag is placed just text to the opening one, instead of wrapping the text:

![image](https://user-images.githubusercontent.com/24541481/182383661-69439d4f-993b-4d4b-ba1d-eaebc6a30f62.png)

As having html in the translation file is not allowed anymore, and injecting it with data-translate-values is not working as expected, I removed it here: 
https://github.com/ovh/manager/pull/7460/commits/6588687921a84edfe9a6c27976980b8d3d5c8856

Reply:
No unfortunately this technics not works when if we separte tags :(.
It's works only if we have same tags in same string like `<b> + $ctrl.myValue + </b>`.